pkg/receive/hashring.go

```diff
@@ -142,19 +157,39 @@ func newKetamaHashring(endpoints []string, sectionsPerNode int, replicationFacto
 	}, nil
 }
 
+func getMinAz(m map[string]int64) int64 {
```

Comment:
I think this function deserves a comment since the input is a generic `map[string]int64`. Does it return the least populated AZ for an endpoint?

Reply:
renamed to
```
func sizeOfLeastOccupiedAZ(azSpread map[string]int64) int64 {
```